Editor/EditorCore/ObjExporter.cs

```diff
@@ -96,6 +96,15 @@ public static bool Export(string name, IEnumerable<Model> models, out string obj
                 return false;
             }
 
+            if (models.Where(m => m.vertices.Any()).Count() < 1)
```

Comment:
Could use `m.vertexCount` here to avoid creating copies of the vertex array.

Reply:
I've just updated this 👍